cmake/utils.cmake

```diff
@@ -161,7 +161,7 @@ function(InstallExternalTarget target_name target_path install_type installed_fi
 
   install(FILES "${output_file_path}"
     TYPE ${install_type}
-    PERMISSIONS OWNER_READ OWNER_EXECUTE
+    PERMISSIONS OWNER_READ OWNER_EXECUTE OWNER_WRITE
```

Comment:
why do we want this?

Reply:
There was an issue with packaging where `strip` could not rewrite the installed file which failed here https://github.com/lifting-bits/remill/actions/runs/6498034215/job/17648485113#step:6:6691
and was resolved by this change here https://github.com/lifting-bits/remill/actions/runs/6498574821/job/17650196369#step:6:6651